### PR TITLE
OSASINFRA-902: Update OpenStack Custom External LB and DNS Docs

### PR DESCRIPTION
**Pull request by iamemilio**
The underlying network architecture has changed a lot since these docs
were initially written. We want to make sure that these docs are accurate
and up to date so that users with complex networking use cases like workers
on a custom subnet and baremetal workers are able to manage their ingress/egress
traffic as needed. More up to date examples and reference information has been added.

We have chosen to omit sections outlining how to replace the internal lb and dns
services since they were inaccurate. We are targeting an upcoming release to handle
these features better given the complexity of our current networking architecture.

### CODE REVIEW

**Comment by maxwelldb** [comment]
@iamemilio Would you like suggestions on this, or is the review request more of an FYI only thing?

**Comment by iamemilio** [comment]
Yes please!

File: docs/user/openstack/README.md
Comment (maxwelldb) [inline docs/user/openstack/README.md:609]:
```suggestion
The simplest way to verify that apps are reachable is to open the OpenShift console in a web browser. 

If you don't have access to a web browser, query the console by using the following `curl` command:
```

**Review by maxwelldb — APPROVED** [review]
Made a number of comments. Feel free to accept/trash/interrogate however you like.

File: docs/user/openstack/README.md
Comment (mandre) [inline docs/user/openstack/README.md:572]: The anchor should be #create-api-and-ingress-dns-records, otherwise the link doesn't render. The sentence ends bizarrely too.
```suggestion
To ensure that your API and apps are accessible through your load balancer, [create or update DNS entries](#create-api-and-ingress-dns-records) for them. To use your new load balancing service for external traffic, make sure the IP address for these DNS entries is the IP address your load balancer:
```

**Comment by iamemilio** [comment]
removing the hold, reviewers can lgtm again when ready

/hold cancel

**Pull request merged**
